packages/sources/trueusd/src/config.ts

```diff
@@ -9,5 +9,6 @@ export const DEFAULT_BASE_URL = 'https://core-api.real-time-attest.trustexplorer
 export const makeConfig = (prefix?: string): Config => {
   const config = Requester.getDefaultConfig(prefix)
   config.api.baseURL = config.api.baseURL || DEFAULT_BASE_URL
+  config.defaultEndpoint = DEFAULT_ENDPOINT
```

Comment:
is this supposed to be part of this PR?

Reply:
Yes, we did this so we could add
```
export const endpointResultPaths = {
  trueusd: 'totalTrust',
}
```
since the `resultPath` is otherwise removed.